src/SourceLink.Git.IntegrationTests/BitbucketGitTests.cs

```diff
@@ -13,7 +13,7 @@ public BitBucketGitTests()
         }
 
         [ConditionalFact(typeof(DotNetSdkAvailable))]
-        public void FullValidation_Https()
+        public void FullValidation_CloudHttps()
```

Comment:
>FullValidation_CloudHttps [](start = 20, length = 25)

This is odd - we say that the default is EnterpriseEdition="true", but the `bitbucket.org` host specified in `src\SourceLink.Bitbucket.Git\build\Microsoft.SourceLink.Bitbucket.Git.props` should be using the Cloud Edition URl format, right?

Shouldn't the `SourceLinkBitbucketGitHost` for `bitbucket.org` have `EnterpriseEdition="false"`?

```
<SourceLinkBitbucketGitHost Include="bitbucket.org" EnterpriseEdition="false" />
```

Reply:
Oh, I see, you're special casing `bitbucket.org` in the code:

```C#
if (!BitbucketCloudHostingUrl.Equals(gitUri.GetHost(), StringComparison.OrdinalIgnoreCase))
```

We can remove that special case and specify the property in the props file directly.

---
In reply to: [283574378](https://github.com/dotnet/sourcelink/pull/246#discussion_r283574378) [](ancestors = 283574378)